tests/taskomatic_tests/test_taskomatic.py

```diff
@@ -64,18 +65,17 @@ def test_main(self, monkeypatch):
             assert job.id in TASKOMATIC_JOBS
 
 
-class TestTaskomaticApp(AsyncHTTPTestCase):
+class TestTaskomaticApp:
     """Test taskomatic App handlers"""
 
-    def get_app(self):
-        return taskomatic.APP
+    pytestmark = pytest.mark.asyncio
```

Comment:
you can decorate class instead of pytestmark if you want this marker to be used just in this class, pytestmark is a legacy way how to do it, decorator is IMHO cleaner https://docs.pytest.org/en/6.2.x/example/markers.html#marking-whole-classes-or-modules